pkg/syncer/syncer.go

```diff
@@ -69,41 +69,51 @@ const SyncDown SyncDirection = "down"
 // SyncUp indicates a syncer watches resources on the target cluster and applies the status to KCP
 const SyncUp SyncDirection = "up"
 
-func StartSyncer(
-	ctx context.Context,
-	upstream, downstream *rest.Config,
-	resources sets.String,
-	kcpClusterName logicalcluster.LogicalCluster,
-	pcluster string,
-	numSyncerThreads int,
-	importPollInterval time.Duration,
-) error {
+// SyncerConfig defines the syncer configuration that is guaranteed to
+// vary across syncer deployments. Capturing these details in a struct
+// simplifies defining these details in test fixture.
+type SyncerConfig struct {
+	UpstreamConfig      *rest.Config
+	DownstreamConfig    *rest.Config
+	ResourcesToSync     sets.String
+	KCPClusterName      logicalcluster.LogicalCluster
```

Comment:
LogicalCluster? KCPClusterName is not a term.

Reply:
I'm not inventing this - it's all over the syncer - so I was using the same name for consistency.

Reply:
Done.